src/scripts/lexmatch-sssom-compare.py

```diff
@@ -54,6 +60,28 @@ def main(verbose: int, quiet: bool):
         logger.setLevel(level=logging.ERROR)
 
 
+@main.command("combine_unmapped_lex_exacts")
+def combine_unmapped_lex_exacts() -> None:
+    file_list = [file for file in listdir(LEXMATCH_DIR) if file.endswith("_lex_exact.tsv")]
+    # Initialize an empty DataFrame to store the merged data
+    merged_lex_exact_df = pd.DataFrame()
+
+    # Concatenate all the files into a single DataFrame
+    for file in file_list:
+        file_path = join(LEXMATCH_DIR, file)
+        df = pd.read_csv(file_path, delimiter="\t")
+        merged_lex_exact_df = pd.concat([merged_lex_exact_df, df])
+
+    merged_lex_exact_df = pd.concat(
+        [pd.DataFrame.from_dict(ROBOT_ROW_DICT, orient="columns"), merged_lex_exact_df],
```

Comment:
Perhaps this is ever so slightly simpler; here's an example of how I concat in the robot row:
```py
ROBOT_TEMPLATE_HEADER = {'mondo_id': 'ID', 'mondo_label': 'LABEL', 'xref': 'A oboInOwl:hasDbXref'}
final_df = pd.concat([pd.DataFrame([ROBOT_TEMPLATE_HEADER]), df])
```